app/src/main/java/com/checkin/app/checkin/manager/fragments/ManagerScheduledLiveOrdersFragment.kt

```diff
@@ -67,6 +67,9 @@ class ManagerScheduledLiveOrdersFragment : BaseFragment(), PreorderTableInteract
                 }
             }
         })
+        viewModel.doneData.observe(this, Observer {
+            // observer needed to ensure "mark food ready" POST request is fulfilled.
+        })
```

Comment:
This observer causes problem for the toggle button present for going live. It usually throws an exception related to lifecycle object.

Reply:
Okay, but use another way to ensure the crash doesn't occur. The livedata needs to be observed atleast once in the fragment.